src/collections/components/CollectionForm/CollectionForm.test.tsx

```diff
@@ -21,6 +21,9 @@ describe('The CollectionForm component', () => {
   let languages: CollectionLanguage[];
   const handleSubmit = jest.fn();
 
+  // This test suite occasionally takes forever to run on a local machine
+  jest.setTimeout(10000);
+
```

Comment:
PRs to a feature branch don't run all the 300+ tests that we have for this repo but I'm running them locally to be sure things don't fall over.